### PR TITLE
continue PR: " docs: improve documentation around gathering telemetry"

### PR DESCRIPTION
**Pull request by ppcano**
This PR continues #33. I wait for a decision whether it should be merged into the same branch/PR. 

(1) - For easier docker setup, this PR adds docker-compose files: `docker-compose-local.yaml` and `docker-compose-cloud.yaml`

(2) - Updated readme instructions

(3) - Could not setup Loki (see [error on this diff](https://gist.github.com/ppcano/2792e26b251b9bc88af9249b31dd8833)). We can try i the Loki integration in a following PR.

(4) - Next, on another PR, I want to include a case to send k6 metrics to Prometheus and visualize with the pre-built dashboards. 

I have tested both docker compose setups and they work well.  PR seems ready - adding also @dgzlopes for review.

### CODE REVIEW

File: contrib/agent-local.river [hunk 1]
```diff
@@ -56,6 +56,10 @@ otelcol.exporter.otlp "default" {
   client {
     // TODO: Replace this with the endpoint for your trace receiver
     endpoint = env("TRACES_ENDPOINT")
+    tls {
```
Comment (ppcano) [inline contrib/agent-local.river:59]: I had to add this. cc @theSuess

File: docker-compose-cloud.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+services:
+  quickpizza:
+    image: ghcr.io/grafana/quickpizza-local:latest
+    ports:
+      - "3333:3333"
+    environment:
+      QUICKPIZZA_OTLP_ENDPOINT: http://agent:4318
+      QUICKPIZZA_TRUST_CLIENT_TRACEID: 1
+      # must be set with an .env file
```
Comment (ppcano) [inline docker-compose-cloud.yaml:11]: Users must configure env variables in an .env file. 

I'll add the proper instructions on the readme if this is approved.

File: contrib/agent-local.river (hunk 1 above)
Comment (theSuess) [reply contrib/agent-local.river:47]: Seems good. Haven't tested this with a local tempo setup, so thanks for checking this out!

**Comment by theSuess** [comment]
The issue regarding the autoconfigure module is probably caused by token permissions. The token needs the `stacks:read` scope. It sounds like this is missing. We'll have to decide if we want to put the extra burden on token creation or have the user collect all the relevant infos ({prom,loki,tempo,pyroscope} url, tenant) themselves

File: cmd/main.go [hunk 3]
```diff
@@ -1,5 +1,8 @@
 package main
 
+import _ "net/http/pprof"
+import _ "github.com/grafana/pyroscope-go/godeltaprof/http/pprof"
```
Comment (theSuess) [inline cmd/main.go:4]: We don't need these imports. Profiling is already enabled using a middleware

**Review by theSuess — CHANGES_REQUESTED** [review]

File: cmd/main.go (hunk 3 above)
Comment (ppcano) [reply cmd/main.go]: corrected by https://github.com/grafana/quickpizza/pull/34/commits/007326344263e042ae40a2a76dbf935dff6d1f1d

Thanks!

**Comment by ppcano** [comment]
> The issue regarding the autoconfigure module is probably caused by token permissions. The token needs the stacks:read scope. It sounds like this is missing.

Yep, that was, thank you! I added minor [comment](https://github.com/grafana/quickpizza/pull/34/commits/884f05a93cd7abd7456373e1775592cab223a0df).

I got confused because I could not find the [GET stack API documentation](https://grafana.com/docs/grafana-cloud/developer-resources/api-reference/cloud-api/#stacks). Should I open an issue?

File: docker-compose-cloud.yaml (hunk 2 above)
Comment (ppcano) [reply docker-compose-cloud.yaml:14]: See -> https://github.com/grafana/quickpizza/tree/pepe/docs/grafana-agent#grafana-cloud

**Review by dgzlopes — APPROVED** [review]

**Review by theSuess — APPROVED** [review]

**Pull request merged**